app/tailwind.css

```diff
@@ -34,15 +38,22 @@
 
 html {
   line-height: 1.5;
+
   /* 1 */
-  -webkit-text-size-adjust: 100%;
+  text-size-adjust: 100%;
+
   /* 2 */
-  -moz-tab-size: 4;
+  tab-size: 4;
+
   /* 3 */
-  -o-tab-size: 4;
-     tab-size: 4;
+  tab-size: 4;
+  tab-size: 4;
+
   /* 3 */
```

Comment:
Can't figure out why `Number` comments are all over the CSS files.
@harshith-venkatesh @dhanush027 What's the use of adding these comments ??

Reply:
I haven't modified the file,but i think it was used to give some reference.